evm-adapters/src/sputnik/cheatcodes/cheatcode_handler.rs

```diff
@@ -95,6 +95,10 @@ impl<'a, 'b, B: Backend, P: PrecompileSet> SputnikExecutor<CheatcodeStackState<'
         U256::from(self.state().metadata().gasometer().gas())
     }
 
+    fn all_logs(&self) -> Vec<String> {
+        self.handler.state().all_logs.clone()
```

Comment:
Just learning: in the case of expected_revert, @gakonst had indicated a preference for using a reference rather than cloning. Would you want to use a reference here, as well?

Reply:
also, in my case, clone actually didn't work for some reason. But I'm sure you know more!